sesame/src/main/scala/org/w3/banana/sesame/SesameOps.scala

```diff
@@ -167,7 +167,7 @@ class SesameOps extends RDFOps[Sesame] with SesameMGraphOps with DefaultURIOps[S
   def isomorphism(left: Sesame#Graph, right: Sesame#Graph): Boolean = {
     val leftNoContext = left.asScala.map(s => makeTriple(s.getSubject, s.getPredicate, s.getObject)).asJava
     val rightNoContext = right.asScala.map(s => makeTriple(s.getSubject, s.getPredicate, s.getObject)).asJava
-    ModelUtil.equals(leftNoContext, rightNoContext)
+    Models.isomorphic(leftNoContext, rightNoContext)
```

Comment:
I have opened [issue 267: new sesame Graph contains contexts](https://github.com/w3c/banana-rdf/issues/267).
I am ok with accepting the changes here on condition that a test is built to check that the context issue I brought up does not appear - which I doubt - and that the issue is treated as REQUIRING a solution.